source/ambient.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module "node-fetch";
+declare module "parse-diff";
+declare module "lodash.includes";
+declare module "lodash.find";
+declare module "jest-runtime";
+declare module "jest-environment-node";
+
+declare module "*/package.json";
```

Comment:
@alloy yes, I usually prefer to avoid `require` to escape hatch for non-typed package as much as I can. Nearly most cases it won't be needed.